tmt/steps/discover/shell.py

```diff
@@ -60,7 +60,7 @@ class TestDescription(
     description: Optional[str] = None
     enabled: bool = True
     order: int = field(
-        # TODO: ugly circular dependency (see tmt.base.DEFAULT_ORDER)
+        # TODO: ugly circular dependency (see tmt.steps.PHASE_ORDER_DEFAULT)
```

Comment:
Perhaps this one would be resolved as well if it's defined under `tmt.steps`?

Reply:
Nice, fixed in 0fb39379

Reply:
Nope, fixed in 8e3586a6